### PR TITLE
HEEDLS-793 Add PRN to manage delegate page

### PR DESCRIPTION
**Pull request by livzorn**
### JIRA link
[_HEEDLS-793_](https://softwiretech.atlassian.net/browse/HEEDLS-793)

### Description
Added PRN field to manage delegate page, with the same format as the My Account page as specified in the ticket - showing the PRN if the delegate has one, "Not yet provided" if they haven't been prompted yet, and "Not professionally registered if they have been prompted.

I also changed the capitalisation of "Professional Registration Number" on the My Account page to match how we display it on all other pages.

### Screenshots
localhost_5001_TrackingSystem_Delegates_254480_View_returnPage=1 (5)
localhost_5001_TrackingSystem_Delegates_254480_View_returnPage=1 (4)
localhost_5001_TrackingSystem_Delegates_254480_View_returnPage=1 (3)

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors.
- [ ] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [x] Manually tested my work with and without JavaScript. Full manual testing guidelines can be found here: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/6703648740/Testing
- [ ] Updated/added documentation in Swiki and/or Readme. Links (if any) are below:
  - [doc_1_here](link_1_here)
- [ ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken.
- [x] Scanned over my own MR to ensure everything is as expected.

### CODE REVIEW

**Review by AlexJacksonDS — APPROVED** [review]

File: DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/ViewDelegate/ViewDelegateViewModel.cs [hunk 1]
```diff
@@ -20,10 +20,15 @@ IEnumerable<DelegateCourseDetails> delegateCourses
             DelegateInfo = new DelegateInfoViewModel(delegateUser, customFields);
             DelegateCourses = delegateCourses.Select(x => new DelegateCourseInfoViewModel(x)).ToList();
             Tags = FilterableTagHelper.GetCurrentTagsForDelegateUser(delegateUser);
+
+            ProfessionalRegistrationNumber = delegateUser.HasBeenPromptedForPrn
+                ? delegateUser.ProfessionalRegistrationNumber ?? "Not professionally registered"
+                : "Not yet provided";
```
Comment (DanBloxham-sw) [inline DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/ViewDelegate/ViewDelegateViewModel.cs:26]: Is there a helper method or a delegateUser property that sets this that could be used on the My Account page as well? Or do they not share enough to be commonised?

**Review by DanBloxham-sw — COMMENTED** [review]
Minor question, but looks good otherwise.

File: DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/ViewDelegate/ViewDelegateViewModel.cs (hunk 1 above)
Comment (livzorn) [reply DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/ViewDelegate/ViewDelegateViewModel.cs]: Good idea, added a method on DelegateUser for it

File: DigitalLearningSolutions.Data/Models/User/DelegateUser.cs [hunk 2]
```diff
@@ -46,5 +48,12 @@ public CentreAnswersData GetCentreAnswersData()
                 Answer6
             );
         }
+
+        public string GetPrnDisplayString()
```
Comment (DanBloxham-sw) [inline DigitalLearningSolutions.Data/Models/User/DelegateUser.cs:52]: Worth putting this as an expression body definition property and making some unit tests for it?

**Review by DanBloxham-sw — COMMENTED** [review]
Think the new method could be improved slightly.

File: DigitalLearningSolutions.Data/Models/User/DelegateUser.cs (hunk 2 above)
Comment (AlexJacksonDS) [reply DigitalLearningSolutions.Data/Models/User/DelegateUser.cs:52]: I'd probably put this in DisplayStringHelper

Comment (livzorn) [reply DigitalLearningSolutions.Data/Models/User/DelegateUser.cs]: Done!

Comment (livzorn) [reply DigitalLearningSolutions.Data/Models/User/DelegateUser.cs]: Ah whoops, I must have not refreshed the page and missed the comment about DisplayStringHelper. Moved the method to there.

**Review by DanBloxham-sw — APPROVED** [review]
Yep, looks good now.

**Pull request merged**